src/Data/ElfEdit/Types.hs

```diff
@@ -375,6 +375,11 @@ pattern PT_HIOS    = ElfSegmentType 0x6fffffff
 -- | Start of OS-specific
 pattern PT_LOPROC :: ElfSegmentType
 pattern PT_LOPROC  = ElfSegmentType 0x70000000
+
+-- | Exception unwinding tables
+pattern PT_ARM_EXIDX :: ElfSegmentType
+pattern PT_ARM_EXIDX  = ElfSegmentType 0x70000001
+
```

Comment:
Does this need to be exported?

Reply:
Yep, see latest version. :smile:

Reply:
Also from Data.ElfEdit (the top level module).  Approved other than that